OJP/OJP_Common.xsd

```diff
@@ -188,6 +188,28 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
+	<xs:complexType name="VehicleFilterStructure">
+		<xs:annotation>
+			<xs:documentation>Filter for Vehicles</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="Exclude" type="xs:boolean" default="true" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Whether Vehicle and TRAIN NUMBERS in list are to include or exclude from search. Default is exclude.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="VehicleRef" type="siri:VehicleRefStructure" minOccurs="0" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>Reference to transport operator</xs:documentation>
```

Comment:
operator?

Reply:
isn't there already is already an "OperatorFilter" (type="OperatorFilterStructure") ?

Reply:
copy/paste error corrected